src/test/resources/rest-api-spec/test/ingest_opennlp/10_basic.yaml

```diff
@@ -8,21 +8,5 @@
         nodes.info: {}
 
     - match:  { nodes.$master.plugins.0.name: ingest-opennlp }
-    - match:  { nodes.$master.ingest.processors.0.type: append }
-    - match:  { nodes.$master.ingest.processors.1.type: convert }
-    - match:  { nodes.$master.ingest.processors.2.type: date }
-    - match:  { nodes.$master.ingest.processors.3.type: date_index_name }
-    - match:  { nodes.$master.ingest.processors.4.type: fail }
-    - match:  { nodes.$master.ingest.processors.5.type: foreach }
-    - match:  { nodes.$master.ingest.processors.6.type: gsub }
-    - match:  { nodes.$master.ingest.processors.7.type: join }
-    - match:  { nodes.$master.ingest.processors.8.type: lowercase }
-    - match:  { nodes.$master.ingest.processors.9.type: opennlp }
-    - match:  { nodes.$master.ingest.processors.10.type: remove }
-    - match:  { nodes.$master.ingest.processors.11.type: rename }
-    - match:  { nodes.$master.ingest.processors.12.type: set }
-    - match:  { nodes.$master.ingest.processors.13.type: sort }
-    - match:  { nodes.$master.ingest.processors.14.type: split }
-    - match:  { nodes.$master.ingest.processors.15.type: trim }
-    - match:  { nodes.$master.ingest.processors.16.type: uppercase }
+    - match:  { nodes.$master.ingest.processors.0.type: opennlp }
```

Comment:
I think this is a result of all the processors moving into a module, `ingest-common`